runtime/caml/incbin.h

```diff
@@ -0,0 +1,368 @@
+/**
```

Comment:
FTR note that this code is (un!)covered by the https://en.m.wikipedia.org/wiki/Unlicense